internal/documentation/jsdoc-to-markdown-v1.js

```diff
@@ -0,0 +1,197 @@
+}
+
+async function htmlToMarkdown(options = {}) {
+	const file = await unified()
+		.use(rehypeParse, {fragment: true})
+		.use(rehypeIgnore)
+		.use(remarkGfm)
+		.use(rehypeVideo)
+		.use(rehypeFormat)
+		.use(rehypeRemark, {
+            document: false,
+            handlers: {
+                table(state, node) {
+                    // This fixes nested tables that happen in the conversion from html to markdown
+                    let html = toHtml(node);
+                    const parsedHTML = new JSDOM(html).window.document.getElementsByClassName("params")[1];
+                    if (parsedHTML !== undefined) {
+                        html = parsedHTML.outerHTML;
+                    }
+                    const result = {type: "html", value: html};
+                    state.patch(node, result);
+                    return result;
+                },
+                a(state, node) {
+                    // This fixes internal linking e.g. from #~myawesomemethod to #myawesomemethod
+                    const result = {type: "html", value: toHtml(node).replaceAll("~", "")};
+                    let href = result.value.match(aElementRegex);
+                    // Filters for a tags that need fixing
+                    // href !== null - Checks if href is null (A element has no href)
+                    // href[1].split("#")[1] != null - Checks if the a element is an internal link
+                    // !href[2].includes("line") - Checks for links to GitHub source code
+                    if (href !== null && href[1].split("#")[1] != null && !href[2].includes("line")) {
+                        const text = href[2];
+                        href = href[1];
+                        result.value =
+                            result.value.replace(href, href.split("#")[0] + "#" + href.split("#")[1].toLowerCase())
+                                .replace(text, text.replace(href.split("#")[1], "") + "#" + href.split("#")[1])
+                                .replace("##", "#");
+                    }
+                    state.patch(node, result);
+                    return result;
+                }
+            }
+        })
+		.use(remarkStringify, {
+            commonmark: true,
+            entities: true
+        })
+		.processSync(options.html);
+	return String(file);
+}
+
+const deadLinks = [];
+const deadLinksCheckPromises = [];
+const excludedFiles = {"index.html": "", "global.html": "", "custom.css": ""};
+const inputDirectory = path.join("dist", "api");
+const outputDirectory = path.join("docs", "api");
+let packageTagName = "https://github.com/UI5/cli/blob/main/packages/";
+
+// 1. Find tag name
+if (process.argv[2] == "gh-pages") {
+    // Read package.json of packages/builder
+    const builderJson = JSON.parse(fs.readFileSync("tmp/packages/@ui5/builder/package.json"));
+    packageTagName = `https://github.com/UI5/cli/blob/cli-v${builderJson["version"]}/packages/`;
```

Comment:
This link is invalid, because the previous versions of UI5 CLI are not within a monorepo, but separate packages and they have separate repositories. There is no `packages` folder, except for the main branch